/**:
  ros__parameters:
    backpropagation:
      backpropagation_gain_u: [-1., -0.5, -0.05, 0., 0.05, 0.5, 1.]
      backpropagation_gain_positive_y: [0., 0., 0., 0.5, 0.9, 1., 1.]
      backpropagation_gain_negative_y: [1., 1., 0.9, 0.5, 0., 0., 0.]
      propagation_gain: 0
